### PR TITLE
Add feature to write an attribute to a node

### LINKED ISSUE #285: How to write attributes from clients
Hi
I'm developing fan control integration, but this cluster don't have any commands to set speed. I would like to set speed attribute but python-matter-server don't support any API to do it.
Similarly, I think we should have `APICommand.WRITE_ATTRIBUTE`
```
@api_command(APICommand.WRITE_ATTRIBUTE)
    async def write_attribute(self, node_id: int, endpoint: int, attribute: ClusterAttributeDescriptor, timedRequestTimeoutMs: int = None, interactionTimeoutMs: int = None):
        """Write a attribute on a target node."""
        LOGGER.info("Write Attributes node_id: %s", node_id)
        attributes = [(endpoint, attribute)]
        return await self.chip_controller.WriteAttribute(
            nodeid=node_id,
            attributes=attributes,
            timedRequestTimeoutMs=timedRequestTimeoutMs,
            interactionTimeoutMs=interactionTimeoutMs,
        )
```

### PR DESCRIPTION
**Pull request by marcelveldt**
Fixes #285

### CODE REVIEW

File: matter_server/client/client.py [hunk 1]
```diff
@@ -231,6 +231,28 @@ async def send_device_command(
             interaction_timeout_ms=interaction_timeout_ms,
         )
 
+    async def write_attribute(
```
Comment (MartinHjelmare) [inline matter_server/client/client.py:234]: What features need to write attributes? Why doesn't all features that does a change have a device command?

Comment (marcelveldt) [reply matter_server/client/client.py:234]: commands that are not predefined in the clusters, manufacturer specific data.
For example setting the mode of a thermostat is not defined as cluster command but thermostats can accept it with writing an attribute

Comment (marcelveldt) [reply matter_server/client/client.py:236]: see chapter 8.9.2.4 of the specs btw

File: matter_server/server/device_controller.py [hunk 2]
```diff
@@ -383,6 +383,24 @@ async def send_device_command(
                 interactionTimeoutMs=interaction_timeout_ms,
             )
 
+    @api_command(APICommand.WRITE_ATTRIBUTE)
+    async def write_attribute(
+        self,
+        node_id: int,
+        attribute_path: str | list[str],
```
Comment (MartinHjelmare) [inline matter_server/server/device_controller.py:390]: Maybe only support a single path? Is it common to write the same value to multiple paths at the same time?

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**